### PR TITLE
Prettyprint and add permid~s

### PR DESCRIPTION
**Pull request by davidfarmer**
Every ptx file has been prettprinted and, and permid~s added to all appropriate tags
(unless some were missed).

It is easy for me to re-run the code if something is not right.

### CODE REVIEW

**Comment by davidfarmer** [comment]
"webwork" tags are ignored by the prettyprinter (I guess I was
scared of them for some reason, but shouldn't have been.)
I'll fix and update the pull request.

File: ptx/exercises/seq-arithgeom.ptx [hunk 1]
```diff
@@ -1,423 +1,535 @@
...
-                36.
-              </p>
-            </li>
-            <li><m>\frac{253 \cdot 36}{2} = 4554</m>.</li>
-          </ol>
-        </p>
-      </solution>
-    </webwork>
+    <setup>
+    <pg-code>Context()->variables->are(n => "Real");
```
Comment (oscarlevin) [inline ptx/exercises/seq-arithgeom.ptx]: While you are fixing webwork things, <pg-code> could use some attention.

File: ptx/exercises/seq-arithgeom.ptx [hunk 2]
```diff
@@ -1,423 +1,535 @@
...
+  <exercise permid="hip">
+    <webwork><statement>
+      <p permid="zOJ">
+        Find <m>x</m> and <m>y</m> such that
+        <m>27,
+        x, y, 1</m> is part of an arithmetic sequence.
+      </p>
 
+      <instruction>
+      <m>x = </m> <var name="'55/3'" width="5"/>, <m>y = </m> <var name="'29/3'" width="5"/>
```
Comment (oscarlevin) [inline ptx/exercises/seq-arithgeom.ptx]: And <instruction> should behave like a <p>; it's content should be indented.

**Comment by davidfarmer** [comment]
See separate pull request.  Close this after comparing and checking that the new one addressed
those shortcomings.

**Comment by oscarlevin** [comment]
Closing this since #66 made this obsolete.

**Pull request closed without merging**